other/catalan.go

```diff
@@ -0,0 +1,20 @@
+package testing101
```

Comment:
When I said proper package names, I meant it in the context of the project. The Go way to do it is that the package name is the folder in which the file is kept.